src/anbox/logger.h

```diff
@@ -52,6 +52,7 @@ class Logger : public DoNotCopyOrMove {
   bool SetSeverityFromString(const std::string &severity);
   virtual void SetSeverity(const Severity& severity) = 0;
   virtual Severity GetSeverity() = 0;
+  virtual void SetLogFile(const std::string& filename){log_file_name = filename;};
```

Comment:
Please follow the code style and add implementation to the actual implementation class.